code/game/objects/items/weapons/twohanded.dm

```diff
@@ -432,3 +432,75 @@
 			M.faction = user.faction.Copy()
 			M.Copy_Parent(user, 100, user.health/2.5, 12, 30)
 			M.GiveTarget(L)
+
+/obj/item/weapon/twohanded/ram
+	name = "kinetic battering ram"
+	desc = "Internal gyroscopes give this ram excellent balance and the necessary force to break doors... or ribs."
+	icon_state = "ram0"
+	force = 10
+	force_wielded = 24
```

Comment:
Because it has ~balance implication~ I guess. It's a downgraded fire axe (can't sharpen or dismembermeme) and it doesn't pierce armor or embed like spears. Shouldn't be an issue since sec already have batons.